### DISCUSSION TITLE
kernel panic not syncing No working init found try passing init =option to kernel in Cloud Stack .4.19  ubuntu 22.04

### DISCUSSION THREAD

**Discussion by chadndu25L** [apache/cloudstack#9466] [Q&A]
I have set up CloudStack on Ubuntu 22.04 OS. I can create a small Debian OS and install it successfully. I have registered the Ubuntu 22.04 ISO, but when I launch an instance using this ISO, I am encountering the following error:

kernel panic - not syncing: No working init found. Try passing `init=` option to kernel

Hardware Name: Apache Software Foundation CloudStack KVM Hypervisor, BIOS 1.15.01 01/01/2024

I am new to the CloudStack environment, so any help with this issue would be greatly appreciated.

Regards
Chandrakanth

**Comment by zap51** [comment]
That shouldn't usually happen. Please describe about your setup like ACS version, primary & secondary storage etc.,

**Comment by DaanHoogland** [comment]
closed this for lack of interaction, please reopen or open a new item if applicable
